js/src/modals/AddAddress/addAddress.js

```diff
@@ -89,7 +89,7 @@ export default class AddAddress extends Component {
         onClick={ this.onAdd }
         ref='addButton'
       />
-    ]);
+    ];
```

Comment:
Not trying to fix the all button layouts here. Wait for modal updates round 2, 3, etc. (Need to work through one-by-one to see what works and what doesn't - help is appreciated there, it is a lot of ground to cover to say the least.)

Key is valuable though.